specs/market-api.yaml

```diff
@@ -106,58 +98,45 @@ paths:
         This implies, that client code should not `unsubscribeOffer` before it has received
         all expected/useful inputs from `collectDemands`.
```

Comment:
`collectDemands` in Offer endpoint. Note another occurrence above not visible in diff